components/Renderers/Writing.tsx

```
@@ -0,0 +1,33 @@
+      const content = await response.text();
+      setText(content || description);
+      setIsLoading(false);
+    };
+    if (fileUrl) getText();
+  }, [description, fileUrl]);
+
+  if (isLoading) return <Skeleton className="min-h-[200px] size-full" />;
+  return (
+    <div
+      className="font-archivo bg-grey-moss-100 p-2 rounded-md text-md !normal-case"
+      dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)